lms/djangoapps/bulk_email/tests/test_email.py

```diff
@@ -268,6 +384,41 @@ def test_unicode_students_send_to_all(self):
             [self.instructor.email] + [s.email for s in self.staff] + [s.email for s in self.students]
         )
 
+    def test_long_course_display_name(self):
```

Comment:
If there is not an existing test that verifies the exact format of the from address when < 320 characters (no truncation), please add one.